packages/material-ui/src/styles/createSpacing.d.ts

```diff
@@ -15,6 +15,6 @@ export interface Spacing {
   ): string;
 }
 
-export type SpacingOptions = number | ((factor: number) => string | number);
+export type SpacingOptions = number | ((factor: number) => string | number) | number[];
```

Comment:
If I understand the implementation that this is just forwarded to `@material-ui/system`, right? Seems like we should type `createUnarySpacing` at some point and use that type in /core.

Reply:
Should we do this change here?

Reply:
I don't feel strongly about it. Up to anybody who's interested in either method.

Reply:
Cool, I suggest one step at the time in this case :). @denys-pavlenko Feel free to send a follow-up. Maybe we should start by breaking down the index.d.ts to be closer to the source
https://github.com/mui-org/material-ui/blob/master/packages/material-ui-system/src/index.d.ts